src/build/__main__.py

```diff
@@ -117,7 +114,13 @@ def main_parser():  # type: () -> argparse.ArgumentParser
         action='store_true',
         help='build a wheel',
     )
-    parser.add_argument('--outdir', '-o', metavar='dist', type=str, default=out, help='output directory')
+    parser.add_argument(
+        '--outdir',
+        '-o',
+        metavar='dir',
```

Comment:
Can we make this `srcdir/dist`?

Reply:
Or rather, `os.pathsep.join('srcdir', 'dist')`.

Reply:
I don't mind either way but it's a little unusual for the [metavar](https://docs.python.org/3/library/argparse.html#metavar) to display the default value for the argument.  This is what the help text looks like right now:

```
 --outdir dir, -o dir  output directory (defaults to {srcdir}/dist)
```

And without `metavar`:

```
  --outdir OUTDIR, -o OUTDIR
                        output directory (defaults to {srcdir}/dist)
```

Reply:
But that information is kind of redundant, isn't it? I usually prefer to put the default value there. And we are doing that in other arguments.

Reply:
I like this as is, aka the metavar not containing the default.

Reply:
> And we are doing that in other arguments.

Not that I can see - there are only two labels that appear in the help text - one for `--outdir` and the other for `--config-setting`: `CONFIG_SETTING` - and I feel like it comes at the cost of clarity, that the default should appear in the label's position.

Reply:
(Arguably `CONFIG_SETTING` should be something like `opt[=val]` instead if we're pursuing descriptive labels.)

Reply:
How do you want to proceed here?

Reply:
I've stated my preference. Since both Bernát and you have the same opinion, I guess go with that. But please fix all other arguments to match this.

Reply:
Does fixing mean lowercasing?  And that'd only be for `--config-setting`, right?